### PR TITLE
✨ Second cut for new component: [amp-access-fewcents] paywall support by Fewcents

### PR DESCRIPTION
**Pull request by mayank-s-dev**
Ref to I2I: #[31854](https://github.com/ampproject/amphtml/issues/31854)

**Description:**
This PR is second cut for the amp-access-fewcents component, you can find the first PR [here](https://github.com/ampproject/amphtml/pull/37096), and this PR includes:
- Authorize API call completion
- Handled the authorize response
- Completing the paywall
- Added design
- Added amp-analytics using segment.

**References:**
We have been following amp-access-laterpay and amp-access-poool for our reference.

- [amp-access](https://github.com/ampproject/amphtml/tree/main/extensions/amp-access)
- AMP code contribution docs: https://github.com/ampproject/amphtml/blob/main/docs/contributing-code.md
- First time setup: https://github.com/ampproject/amphtml/blob/main/docs/getting-started-e2e.md
- Building an extension: https://github.com/ampproject/amphtml/blob/main/docs/building-an-amp-extension.md
- [amp-analytics](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/configure-analytics/analytics_basics/?format=websites#use-amp-pixel-or-amp-analytics)
- Segment AMP : https://segment.com/docs/connections/sources/catalog/libraries/mobile/amp/
- [Style and CSS](https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/?format=websites)

### CODE REVIEW

**Comment by mayank-s-dev** [comment]
@calebcordry @powerivq 
Hey Guys, I have worked on the second part of the extension and looking forward for your reviews, Let me know if I could be of any help during the process.

File: extensions/amp-access-fewcents/0.1/fewcents-impl.js
Comment (powerivq) [inline extensions/amp-access-fewcents/0.1/fewcents-impl.js:149]: Is everything properly escaped (especially URL arguments)?

File: examples/amp-access-fewcents.html
Comment (mayank-s-dev) [reply examples/amp-access-fewcents.html:2]: Updated.

File: extensions/amp-access-fewcents/0.1/fewcents-impl.js
Comment (mayank-s-dev) [reply extensions/amp-access-fewcents/0.1/fewcents-impl.js:149]: Fixed.

**Comment by mayank-s-dev** [comment]
@powerivq The changes which you asked have been fixed.

File: extensions/amp-access-fewcents/0.1/test/test-amp-access-fewcents.js
Comment (powerivq) [inline extensions/amp-access-fewcents/0.1/test/test-amp-access-fewcents.js:28]: Should the URL be `https://wallet.hounds.fewcents.co/?xxxx`? Also on 2 other occasions in this PR. The backslash should be required in URL. Maybe many browsers can work it fine, but still better to follow the standard.

**Review by powerivq — CHANGES_REQUESTED** [review]

File: examples/amp-access-fewcents.html
Comment (powerivq) [reply examples/amp-access-fewcents.html:2]: Not pushed yet?

File: extensions/amp-access-fewcents/0.1/test/test-amp-access-fewcents.js
Comment (powerivq) [inline extensions/amp-access-fewcents/0.1/test/test-amp-access-fewcents.js:36]: Ditto

File: extensions/amp-access-fewcents/0.1/test/test-amp-access-fewcents.js
Comment (powerivq) [inline extensions/amp-access-fewcents/0.1/test/test-amp-access-fewcents.js:288]: Ditto

**Review by powerivq — CHANGES_REQUESTED** [review]

File: examples/amp-access-fewcents.html
Comment (mayank-s-dev) [reply examples/amp-access-fewcents.html]: Fixed and pushed. Earlier I changed for other files.

File: extensions/amp-access-fewcents/0.1/test/test-amp-access-fewcents.js
Comment (mayank-s-dev) [reply extensions/amp-access-fewcents/0.1/test/test-amp-access-fewcents.js]: Fixed.

File: extensions/amp-access-fewcents/0.1/test/test-amp-access-fewcents.js
Comment (mayank-s-dev) [reply extensions/amp-access-fewcents/0.1/test/test-amp-access-fewcents.js]: Fixed.

File: extensions/amp-access-fewcents/0.1/test/test-amp-access-fewcents.js
Comment (mayank-s-dev) [reply extensions/amp-access-fewcents/0.1/test/test-amp-access-fewcents.js]: Fixed.

**Comment by mayank-s-dev** [comment]
@powerivq I have made the changes you asked.

**Review by MichaelRybak — APPROVED** [review]
Approving changes in validator-amp-access-fewcents.html and .out. 

Please wait for approval from @powerivq.

**Comment by mayank-s-dev** [comment]
> Approving changes in validator-amp-access-fewcents.html and .out.
> 
> Please wait for approval from @powerivq.

Thank you @MichaelRybak for your approval.

**Review by powerivq — APPROVED** [review]

**Pull request merged**
